src/cache/settings.rs

```diff
@@ -10,15 +12,35 @@
 /// let mut settings = CacheSettings::new();
 /// settings.max_messages(10);
 /// ```
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug)]
 pub struct Settings {
     /// The maximum number of messages to store in a channel's message cache.
     ///
     /// Defaults to 0.
     pub max_messages: usize,
+
+    /// The duration that the serenity will and require a write lock on the
```

Comment:
Change to `The amount of milliseconds cache updates will try to acquire write-locks.`.